tests/test_regrid.py

```diff
@@ -195,3 +195,23 @@ def test_out_of_bounds():
     output = regrid(data)
 
     assert torch.all(torch.isnan(output))
+
+
+@pytest.mark.parametrize("k", [1, 2, 3])
+def test_NearestNeighborInterpolator(k):
+    n = 10000
+    torch.manual_seed(0)
+    lon = torch.rand(n) * 360
+    lat = torch.rand(n) * 180 - 90
+
+    lond = torch.rand(n) * 360
```

Comment:
Would also be good to test with a different number of source and destination query points

Reply:
done